tools/coverage-reports/package.yaml

```diff
@@ -0,0 +1,36 @@
+name:                bazel-coverage-report-renderer
+version:             0.1.0
+github:              "tweag/rules_haskell"
+license:             Apache-2.0
+author:              "Tweag I/O Limited"
+maintainer:          "m@tweag.io"
+copyright:           "2019 Tweag I/O Limited"
+
+extra-source-files:
+- LICENSE
+- package.yaml
+- README.md
+- stack.yaml
+
+synopsis:            Generate coverage report from Rules_Haskell bazel coverage output. See README at <https://github.com/tweag/rules_haskell/blob/master/tools/coverage-reports/README.md>
```

Comment:
You’re distributing the `README.md` with the source, so this link should not be needed.